packages/contract/lib/contract/index.js

```diff
@@ -156,6 +157,10 @@ if (typeof Web3 === "object" && Object.keys(Web3).length === 0) {
 
   Contract._constructorMethods = constructorMethods(Contract);
 
+  Contract.prototype[util.inspect.custom] = function (/* _depth , _options */) {
+    return `Contract at: ${this.address}`;
+  };
+
```

Comment:
This might be controversial. I did this to make the REPL experience better because the giant wall of text to display a contract instance is a massive wall of text, pages long.

Reply:
I actually don't understand what this is doing. I assume from what you said it has something to do with providing custom output when logging contracts. Can you explain to me a little bit how this works?

Reply:
When an object is evaluated in the REPL it will be inspected. For example, when you get the deployed instance for a contract (`meta = await MetaCoin.deployed()`), that contract instance is inspected, and the REPL dumps a wall of text to represent the instance object and its numerous properties. This alters the representation to be a single line:  `Contract at: ..address..`

Reply:
Ok so `util.inspect.custom` is the symbol property on prototype that defines the result of inspecting the object in question?

Reply:
yes

Reply:
@eggplantzzz I didn't think of the implication to webpacked @truffle/contract :/. I'll investigate Monday.

Reply:
> * Let `a = Telephone.new(1)` and `b = Telephone.new(1)`, they both print `Contract at: ...`! Pointing out whether or not it's a Promise can avoid potential confusion.

> * Since we're changing this symbol, maybe there's more basic information other than just the address that might be useful? ContractName, methods, etc. That way it's not a wall of text and you still get only the info you'll likely need.

I've never had to inspect contract properties beyond `abi`, though many contracts' `abi` could be multi-pagers. What exactly should we present?

> * What if someone wants the default inspector? Like a `contract.asObject` to expose that? Maybe this is already in place via another interface so please correct me if I'm wrong.

Good question! Maybe `delete Contract.prototype[Symbol.for("nodejs.util.inspect.custom")]`, or `console.log(util.inspect(a, {customInspect: false}))`. I'd have to test the former when I get to a computer. 

What do people think? Should we remove this change, and let users monkey patch themselves? Leave it in, or PR separately and wait for user feedback for guidance? @eggplantzzz, @cliffoo, @sukanyaparashar, @fainashalts, @dongmingh, @lsqproduction?

Reply:
Removing this functionality because, as @cliffoo pointed out, it is not well-defined. I'll consider the feedback and put up another PR.